lib/kb_Metrics/Util.py

```diff
@@ -30,10 +31,15 @@ def _convert_to_datetime(dt):
     else:
         raise ValueError('Cannot convert {} to datetime'.format(dt))
 
+
 def _partition_by_keys(src_list, idKeys, dataKeys):
```

Comment:
Add tests to funcs defined in this module.

Reply:
is this comment addressed?

Reply:
Being addressed...need assemble the testing data sets first.